app/javascript/netlify-app.js

```diff
@@ -9,6 +9,14 @@ class NihGateway extends GitGatewayBackend {
   }
 }
 
+[].slice
+  .call(
```

Comment:
`Function.call()` executes a function using the first argument as `this` inside the function. `querySelectorAll` returns a `NodeList`, which is not _technically_ an Array--you can't iterate over its contents using `.forEach` in IE 11. `[].slice.call(<some nodelist>)` converts the NodeList to an Array so you can iterate over it. There are a couple of other ways this could be done:

1. `Array.from(document.querySelectorAll(...)).forEach`, which is clearer, but doesn't work in IE 11 
2. `for (let link of documentQuerySelectorAll(...))`, which also doesn't work in IE 11
3. `for (let i =0; i < nodelist.length; i++)`-style

It's mostly a matter of taste, I guess--I think I prefer `.forEach()` style iteration over `for` statements, but not super strongly.

Reply:
Anyway, ✨ javascript ✨

Reply:
Thanks!

> which is not _technically_ an Array

that absolutely would have tripped me up if I had tried to do this. Also, I agree with preferring `forEach` for array iteration.